### ISSUE TITLE
Do we need to watch our tone more

### ISSUE THREAD

**Issue by magisch** [SO-Close-Vote-Reviewers/SO-Close-Vote-Reviewers.github.io#28]
Lately i've been seeing a lot of comments that read like "Question is just trash" or "OP knows nothing".

Now I admit I've been guilty of this from time to time myself. The problem here isn't strictly the language (not in and of itself appropiate) but I worry that it paints a very bleak picture and does us a great disservice if a question op whose question we closed comes here to engage in civil discussion, only to find that a room of people tacitly made fun of them and their question above.

I know that we don't hate fun, and I know it's hard to follow, but could we more strictly focus on content issues when explaining / commenting on CV-PLS requests?

**Comment by rschrieken** [comment]
So the rule: moderate the post, not the user is failing then? If those rules don't work then maybe we should get rid of them as they have no purpose in guiding the regulars.

**Comment by magisch** [comment]
@rschrieken Somewhat. It's usually probably meant as harmless banter, but someone unfamiliar with the process or OP of the topic reading derogatory comments about the post/poster under a CV pls request may look at that as a voting ring ganging up on people and then mocking them in chat. 

Not only focused on users, but sometimes it is, and even when it's not, things like "Idiot question" or "Useless trash" or "retarded question" would make any OP coming into our room immediately feel unwelcome and treated with hostility.

What I mean is witty banter about how bad the objects of our closure are could make us look _very_ bad to onlookers and destroy opportunities to help OPs who may come to us to understand and learn why we closed something.

**Comment by Tyler-H** [comment]
I think this is pretty important. If we can't follow Rule #2 of SO and "Be Nice" then we have bigger problems than closing questions.

**Comment by Cerbrus** [comment]
Imo, _"Question is just trash"_ is fine. _"OP knows nothing"_ isn't.

There are some question out there that really _are_ trash. I don't remember seeing comments like that about questions that were even remotely salvage-able.

**Comment by Tiny-Giant** [comment]
The reason for close vote requests should be one or more valid applicable close reasons. There should be exactly 0 occurrences of "Question is just trash" or "OP knows nothing" in close vote request reasons as neither of those are valid close reasons.

**Comment by Cerbrus** [comment]
@Tiny-Giant I think this is about statements like that in general. I'm sure we all agree that it's an abysmal cv-plz reason.

**Comment by Tyler-H** [comment]
It's not an 'abysmal' cv-pls reason; it's a disallowed one. it's an invalid one. As in, should never be used to describe a question anyway, because it's unnecessarily rude. We don't have to shower questions with fake praise, but we should remain cognizant of the fact that there is a person on the other side of the computer screen asking for help. If they see someone in our room describing their question as 'trash', it's going to discourage them and/or piss them off. 

There's also no point or constructive reason to describe a question that way in the SOCVR room, which is about removing and moderating off-topic content from SO. We should describe questions as "needing a lot of help" rather than "trash", if we must describe our opinion on their quality (which really isn't necessary in the first place). 

I shouldn't have to repeat myself here, but Rule 2 of Stack Overflow is "Be Nice". I wouldn't expect anyone to sit down and work 1:1 with a person to get a question from -8 and closed to +1 and open/answered, but at the very least don't denigrate them.

**Comment by Tiny-Giant** [comment]
I agree with Tyler. I think that any reference to a specific question should exclude negative comments such as this. Saying a question is trash is different than saying it lacks necessary information required to be on-topic, or that the formatting is sub-par, etc. There are valid observations that can be made about a post without being—for lack of a better word—offensive. 

All of those observations are actually useful as well, whereas describing the question solely as trash offers no guidance to the user on how to improve their question. Provided, there are unsalvageable questions that cannot be improved enough to be on-topic, but the word "unsalvageable" would be much more appropriate than "trash"

The point is that we don't need foul language to moderate content effectively, and the use of foul language is actively detrimental to the room's goals.

**Comment by arp242** [comment]
On chat and meta I always try to keep in the back of my mind that the person I'm talking about might read it (which is indeed a possibility).

It's easy to get into the wrong sort of attitude when talking *about* people, rather than *with* people – I'm certainly susceptible to this at times – and found this helps keep up a constructive and useful attitude.

You wouldn't say "you question is trash" or "you know nothing" when you're talking *with* someone, and we shouldn't when we're talking *about* someone.

**Comment by Tunaki** [comment]
[Bookmark](http://chat.stackoverflow.com/rooms/108179/conversation/2017-january-topic-1)

Stick to the prescribed CV reasons when making requests, don't use deconstructive language when describing a question. We already require members to act professionally and with a high standard.
